pitop/labs/__init__.py

```diff
@@ -0,0 +1,10 @@
+from .web.webserver import WebServer, create_app
+from .web.webcontroller import WebController, RoverWebController
+from .web.blueprints import (
+    RoverControllerBlueprint,
+    BaseBlueprint,
+    ComponentsBlueprint,
```

Comment:
Might be worth changing this name to `WebComponentsBlueprint` to differentiate it from the `components` syntax used for physical robotics components - I initially thought it was something to do with controlling those

Reply:
Yeah I like that actually 👍

Reply:
Done :)